sagemaker-training-compiler/huggingface/tensorflow_tune_batch_size/finding_max_batch_size_for_model_training.ipynb

```diff
@@ -0,0 +1,387 @@
+{
```

Comment:
<p>To define the acronym, the first time you mention Hugging Face, say <code>Hugging Face (HF)</code>. ﻿</p><p> </p><p>Consistently use a space, <code>Hugging Face</code>  , which is the official spelling. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3275/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
Fixed